### DISCUSSION TITLE
v3.11.0

### DISCUSSION THREAD

**Discussion by oxy** [coder/code-server#3779] [Announcements]
VS Code v1.57.1

Upgrading is as easy as installing the new version over the old one. code-server
maintains all user data in `~/.local/share/code-server` so that it is preserved in between
installations.
### New Features

⭐ Upgraded VSCode to v1.57

- feat(vscode): upgrade to version 1.57.1 #3544 @oxy @code-asher

### Bug Fixes

⭐ Fix logout for those behind a reverse proxy

- Fix logout when using a base path (#3608) #3611 @code-asher 

### Documentation

⭐ Lots of reorganizing to make them easier to refer, and some new information!

- docs: add Pomerium #3424 @desimone
- docs: fix confusing sentence in pull requests section #3460 @shiv-tyagi
- docs: remove toc from changelog @oxy @jsjoeio
- docs(MAINTAINING): add information about CHANGELOG #3467 @jsjoeio
- docs: move release process to MAINTAINING.md #3441 @oxy @Prashant168
- docs: format 'Caddy' from guide.md @PisecesPeng

### Development

⭐ Move to Node v14, use Argon2 for pw hashing, and CI work :construction_worker:

- chore: cross-compile docker images with buildx #3166 @oxy
- chore: update node to v14 #3458 @oxy
- chore: update .gitignore #3557 @cuining
- fix: use sufficient computational effort for password hash #3422 @jsjoeio
- docs(CONTRIBUTING): add section on testing #3629 @jsjoeio

### Development

- fix(publish): update cdrci fork in brew-bump.sh #3468 @jsjoeio
- chore(dev): migrate away from parcel #3578 @jsjoeio

Cheers! 🍻

<hr /><em>This discussion was created from the release <a href='https://github.com/cdr/code-server/releases/tag/v3.11.0'>v3.11.0</a>.</em>

**Comment by jsjoeio** [comment]
Head's up — something in GitHub Actions broke with Homebrew so I'm manually publishing now

**Reply by jsjoeio** [reply]
Follow this PR to know when the version has been published on Homebrew: https://github.com/Homebrew/homebrew-core/pull/81274

**Reply by lg-nineoneninethreenine** [reply]
dude its closed

**Reply by lg-nineoneninethreenine** [reply]
what happened? bot closed it automatically

**Reply by jsjoeio** [reply]
That means it should be updated. That's how their PR review process works 🤷‍♂️

**Reply by lg-nineoneninethreenine** [reply]
Never knew it. Thanks for that one nugget of knowledge :)

**Comment by bonanitech** [comment]
When is the container version going to be released?

**Reply by jsjoeio** [reply]
We're currently troubleshooting issues with the Docker release. cc @oxy @code-asher who can share more on the situation

**Reply by rbray89** [reply]
> We're currently troubleshooting issues with the Docker release. cc @oxy @code-asher who can share more on the situation

Is there an ETA on a fix?

**Reply by jsjoeio** [reply]
I wish I could give a specific ETA. All I can say is it's @oxy #1 priority this week (and he works on code-server part-time maybe 15-20hrs/week) so hopefully soon.

**Reply by oxy** [reply]
We recently switched to `buildx` to build docker images; and for some reason, importing the images exported by buildx has been failing. I'm trying to investigate what's causing the issue, and also looking to see if I can publish directly from buildx instead of exporting and re-importing in an attempt to fix the issue.

I think I should be able to fix it by tomorrow or the day after :crossed_fingers:

**Reply by jsjoeio** [reply]
Related: https://github.com/cdr/code-server/issues/3814

**Reply by jsjoeio** [reply]
![image](https://user-images.githubusercontent.com/3806031/127701732-76248957-34fb-4d35-aeb4-f1a7f73055b2.png)

@code-asher has updated the tags. Thanks for your patience!
